lib/task-data/schemas/install-coreos.json

```diff
@@ -40,10 +33,16 @@
             "$ref": "types-installos.json#/definitions/UserPassword"
         },
         "ignitionScriptUri": {
-            "$ref": "#/definitions/IgnitionScriptUri"
+            "$ref": "types-installos.json#/definitions/IgnitionScriptUri"
         },
         "progressMilestones": {
             "$ref": "types-installos.json#/definitions/ProgressMilestones"
+        },
+        "vaultToken": {
+            "$ref": "types-installos.json#/definitions/VaultToken"
```

Comment:
Per your feedback @yyscamper, I have moved **IgnitionScriptUri** back to _install-coreos.json_ as it is indeed CoreOS-specific. I believe that **vaultToken** and **grubLinuxAppend** are universally useful, so I've left them in _types-installos.json_.
Please let me know if there are any other changes you'd like to see. Thanks!